ewkcoffea/modules/selection_wwz.py

```diff
@@ -116,6 +151,24 @@
             "trg_lst" : ["Mu8_TrkIsoVVL_Ele23_CaloIdL_TrackIdL_IsoVL_DZ"],
             "offline_thresholds" : [25.0,10.0],
         },
+    },
```

Comment:
These thresholds will probably need to be updated, right? The effect of these are small, so probably we can move forward for now, but I think we will need to revisit it (since just scanning through the trigger lists it seems like these simple values for each dataset are not falling within the ranges that are listed).

Reply:
So let's remove this too if it's not being used